nameko/testing/pytest.py

```diff
@@ -265,7 +265,7 @@ def test_bar(rabbit_config, container_factory):
 
     reorder_fixtures = ('container_factory', 'runner_factory', 'rabbit_config')
     for fixture in reorder_fixtures:
-        if fixture in request.funcargnames:
+        if fixture in request.fixturenames:
             # getfuncargvalue is renamed to getfixturevalue in pytest 3.0
```

Comment:
Pytest 3.0 was released in 2016, which seems plenty old enough. Let's commit to requiring >3.0 and then we could remove this conditional as well

Reply:
I have pytest-6.0.1, still getting the same warning as stated by adamchainz.
please help me @mattbennett .

Reply:
> I have pytest-6.0.1, still getting the same warning as stated by adamchainz.
> please help me @mattbennett .

I'm getting the same running PyTest v 6.0.1. I wonder why your comment was marked as spam.